<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="calculate_btn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>430</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="num1">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>120</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="num2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>260</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>100</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>First Number</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>240</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Second Number</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="result_lbl">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>340</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>320</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Result</string>
    </property>
   </widget>
   <widget class="QComboBox" name="operators">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>190</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>+</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>/</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>%</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>*</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="invalid_lbl">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>45</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSimple_Calculator">
    <property name="title">
     <string>Simple Calculator</string>
    </property>
   </widget>
   <addaction name="menuSimple_Calculator"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
